### DISCUSSION TITLE
Install Option: Add winget as an install option

### DISCUSSION THREAD

**Discussion by Zalmez** [ppy/osu#15007] [Ideas]
Just an idea, I know there's already a lot of options for installs so is super low prio, not even sure if worthy of a discussion, but I thought I'd bring it up anyway

Winget is microsoft's package manger for windows that does come natively with Windows 11 and as an installable version for windows 10 (for now, seems like winget will be added into windows 10 eventually)

**Comment by bdach** [comment]
We're not interested in adding support for any of the hundreds of package managers out there. The binaries are the only official means of distribution and that goes for all platforms.
